README.md

```diff
@@ -6,7 +6,7 @@ Samples that use the [Google Maps Android API v2](https://developers.google.com/
 This repo contains the following sample:
 
 1. [ApiDemos](https://github.com/googlemaps/android-samples/tree/master/ApiDemos): A collection of small demos showing most features of the Google Maps Android API. (This was formally part of the Google Play Services SDK.)
-1. [BasicWearMap](https://github.com/googlemaps/android-samples/tree/master/BasicWearMap):
+1. [AndroidWearMap](https://github.com/googlemaps/android-samples/tree/master/AndroidWearMap):
```

Comment:
Both of these references should be using repo-relative links.

e.g.
```
 1. [ApiDemos](ApiDemos)
 1. [AndroidWearMap](AndroidWearMap)
```

Would you mind fixing both?